test.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+import winnowing
+
+
+class TestWinnowing(unittest.TestCase):
+    # testing for R language
+    def test1(self):
+        check = winnowing.Winnowing(file1= "/Users/aryangupta/Desktop/IIT_Bombay/MOSS-winnowing-seqMatcher/tests/test6.r",file2="/Users/aryangupta/Desktop/IIT_Bombay/MOSS-winnowing-seqMatcher/tests/test7.r",lang="r")
+        self.assertEqual(check.jaccardCheck(),0.9130434782608695)
+        self.assertEqual(check.plagiarismRate(),0.9130434782608695)
+
+    # testing for Python language
+    def test2(self):
+        check = winnowing.Winnowing(file1= "/Users/aryangupta/Desktop/IIT_Bombay/MOSS-winnowing-seqMatcher/tests/test2.py",file2="/Users/aryangupta/Desktop/IIT_Bombay/MOSS-winnowing-seqMatcher/tests/test3.py",lang="py")
+        self.assertEqual(check.jaccardCheck(),0.5669642857142857)
+        self.assertEqual(check.plagiarismRate(),0.5669642857142857)
```

Comment:
Just round the values and check. Do not check like this.

Reply:
done